src/spec-node/devContainersSpecCLI.ts

```diff
@@ -82,6 +83,16 @@ const mountRegex = /^type=(bind|volume),source=([^,]+),target=([^,]+)(?:,externa
 
 })().catch(console.error);
 
+function findWorkspaceFolder(): string | undefined {
```

Comment:
Wouldn't it be sufficient to just make `.` the default for `--workspace-folder`?